File: languages/java/gladiator.java
import java.io.*;
public class gladiator {
    public static void main(String[] a) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String s = br.readLine();
        String out = "";
        if (s.contains("\"discipline\":\"construct\"")) {
  if (s.contains("\"arena\":\"cheeks\"")) out = "()";
  else if (s.contains("\"arena\":\"ears\"")) out = "/\\\\/\\\\";
  else if (s.contains("\"arena\":\"eyes\"")) out = "oo";
  else if (s.contains("\"arena\":\"geometry\"")) out = "";
  else if (s.contains("\"arena\":\"head_top\"")) out = "_";
  else if (s.contains("\"arena\":\"mouth\"")) out = "^";
  else if (s.contains("\"arena\":\"newlines\"")) out = "";
  else if (s.contains("\"arena\":\"nose\"")) out = ".";
  else if (s.contains("\"arena\":\"padding\"")) out = "      ";
  else if (s.contains("\"arena\":\"whiskers\"")) out = "><";
            System.out.print("{\"ok\":true,\"output\":\"" + out + "\"}");
        } else {
            boolean ok = s.contains("\"candidate\":\" /\\\\_/\\\\\\n( o.o )\\n > ^ <\"");
            System.out.print(ok ? "{\"ok\":true,\"output\":\"" + "es un gato" + "\"}"
                                : "{\"ok\":false,\"output\":\"" + "no es el gato canonico" + "\"}");
        }
    }
}

File: languages/swift/gladiator.swift
let s = readLine() ?? ""
var out = ""
if s.contains("\"discipline\":\"construct\"") {
  if s.contains("\"arena\":\"cheeks\"") { out = "()" }
  if s.contains("\"arena\":\"ears\"") { out = "/\\\\/\\\\" }
  if s.contains("\"arena\":\"eyes\"") { out = "oo" }
  if s.contains("\"arena\":\"geometry\"") { out = "" }
  if s.contains("\"arena\":\"head_top\"") { out = "_" }
  if s.contains("\"arena\":\"mouth\"") { out = "^" }
  if s.contains("\"arena\":\"newlines\"") { out = "" }
  if s.contains("\"arena\":\"nose\"") { out = "." }
  if s.contains("\"arena\":\"padding\"") { out = "      " }
  if s.contains("\"arena\":\"whiskers\"") { out = "><" }
  print("{\"ok\":true,\"output\":\"" + out + "\"}", terminator: "")
} else {
  let ok = s.contains("\"candidate\":\" /\\\\_/\\\\\\n( o.o )\\n > ^ <\"")
  let v = ok ? "es un gato" : "no es el gato canonico"
  print("{\"ok\":" + (ok ? "true" : "false") + ",\"output\":\"" + v + "\"}", terminator: "")
}
